src/main/java/randoop/generation/ObjectPool.java

```diff
@@ -0,0 +1,128 @@
+package randoop.generation;
+
+import static randoop.util.EquivalenceChecker.equivalentTypes;
+
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Set;
+import randoop.DummyVisitor;
+import randoop.ExecutionOutcome;
+import randoop.NormalExecution;
+import randoop.sequence.ExecutableSequence;
+import randoop.sequence.Sequence;
+import randoop.test.DummyCheckGenerator;
+import randoop.types.Type;
+import randoop.util.ListOfLists;
+import randoop.util.SimpleArrayList;
+
+/**
```

Comment:
Randoop already has a class `SequenceCollection` that implements a pool.  How is this class different, and why is it needed?